src/components/EnvelopeCard.js

```diff
@@ -84,57 +84,68 @@ class EnvelopeView extends PureComponent {
 					>
 						<Text style={{ color: 'white', fontWeight: '700' }}>SIGNED</Text>
 					</View>
-					<Button
-							ref={ref => (this.copyButton = ref)}
-							foregroundColor={'white'}
-							backgroundColor={'#454545'}
-							successColor={'#4cd964'}
-							errorColor={'#ff3b30'}
-							errorIconColor={'white'}
-							successIconColor={'white'}
-							shakeOnError={true}
+					<View style={{ flexDirection: 'row', justifyContent: 'center', marginTop: 24 }}>
```

Comment:
Added Signed button so the user now can have multiple signs in the same transaction